src/core_plugins/i18n/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "i18n",
+  "version": "1.0.0"
```

Comment:
We made an update to how versions of "core_plugins" work, the version can now be set to "kibana" to mirror kibana's own version (see the other core_plugins for examples)